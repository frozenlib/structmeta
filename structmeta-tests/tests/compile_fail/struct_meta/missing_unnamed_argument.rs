#[structmeta_tests::attr_required_unnamed_param2(10)]
fn my_func() {}

fn main() {}
