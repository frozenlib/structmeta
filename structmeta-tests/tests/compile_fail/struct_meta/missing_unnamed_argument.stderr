error: expected least 2 arguments but 1 argument was supplied
 --> $DIR/missing_unnamed_argument.rs:1:1
  |
1 | #[structmeta_tests::attr_requred_unnamed_param2(10)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
